frontend/src/js/small-tab-navigation/HoverNavigatable.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "@emotion/styled";
+import { ReactNode, useState } from "react";
+import { useDrop } from "react-dnd";
+
+import { DNDType } from "../common/constants/dndTypes";
+
+interface PropsT {
+  triggerNavigate: () => void;
+  children: ReactNode;
+}
+
+const Root = styled("div")<{
+  isOver?: boolean;
+}>`
+  border: ${({ theme, isOver }) =>
+    isOver ? `1px solid ${theme.col.grayMediumLight}` : "none"};
```

Comment:
Using a border for highlight makes the rest of the layout jump. How about using a light gray background color instead?

Reply:
I have just implemented that using very light gray background. Please let me know what you think